tests/commerce_integration/TEC/Tickets/Commerce/Partials/Checkout/Cart/__snapshots__/CartTest__test_should_render_cart__0.snapshot.html

```diff
@@ -11,14 +11,17 @@ <h4 class="tribe-common-h4 tribe-common-h--alt tribe-tickets__commerce-checkout-
 	<div class="tribe-tickets__commerce-checkout-cart-items">
 					</div>
 
-	<footer  class="tribe-tickets__commerce-checkout-cart-footer tribe-common-b1" >
-
-	<div class="tribe-tickets__commerce-checkout-cart-footer-quantity">
+	
+<footer  class="tribe-tickets__commerce-checkout-cart-footer tribe-common-b1" >
+	
+<div class="tribe-tickets__commerce-checkout-cart-footer-order-modifier-fees">
+	<ul>
+			</ul>
+</div>
```

Comment:
Same as above. I would expcet these snapshots to pass without a need of an update. To ensure we remain backwards compatible